### ISSUE TITLE
Generating Geometry not working

### ISSUE THREAD

**Issue by dariochianetta** [opensourceBIM/BIMserver#1147]
Hi, 
during the "Generating Geometry"  process I have the following errors:

~~~~
16:26:38 ERROR [GeometryRunner]: Forcefully terminated IfcOpenShell process                                       (IfcGeomServerClient.java:724) 
16:26:38 ERROR [GeometryRunner]:                                                                                  (GeometryRunner.java:847) 
org.bimserver.plugins.renderengine.RenderEngineException: java.io.EOFException
	at org.ifcopenshell.IfcGeomServerClient.initialize(IfcGeomServerClient.java:261)
	at org.ifcopenshell.IfcGeomServerClient.loadModel(IfcGeomServerClient.java:281)
	at org.ifcopenshell.IfcOpenShellModel.<init>(IfcOpenShellModel.java:84)
	at org.ifcopenshell.IfcOpenShellEngine.openModel(IfcOpenShellEngine.java:94)
	at org.bimserver.renderengine.pooled.AlwaysClosingRenderEngineWrapper.openModel(AlwaysClosingRenderEngineWrapper.java:32)
	at org.bimserver.geometry.GeometryRunner.run(GeometryRunner.java:197)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.EOFException: null
	at com.google.common.io.LittleEndianDataInputStream.readAndCheckByte(LittleEndianDataInputStream.java:232)
	at com.google.common.io.LittleEndianDataInputStream.readInt(LittleEndianDataInputStream.java:115)
	at org.ifcopenshell.IfcGeomServerClient.initialize(IfcGeomServerClient.java:249)
	... 10 common frames omitted
16:26:39 ERROR [GeometryRunner]: Forcefully terminated IfcOpenShell process                                       (IfcGeomServerClient.java:724) 
16:26:39 ERROR [GeometryRunner]:                                                                                  (GeometryRunner.java:847) 
org.bimserver.plugins.renderengine.RenderEngineException: java.io.EOFException
	at org.ifcopenshell.IfcGeomServerClient.initialize(IfcGeomServerClient.java:261)
	at org.ifcopenshell.IfcGeomServerClient.loadModel(IfcGeomServerClient.java:281)
	at org.ifcopenshell.IfcOpenShellModel.<init>(IfcOpenShellModel.java:84)
	at org.ifcopenshell.IfcOpenShellEngine.openModel(IfcOpenShellEngine.java:94)
	at org.bimserver.renderengine.pooled.AlwaysClosingRenderEngineWrapper.openModel(AlwaysClosingRenderEngineWrapper.java:32)
	at org.bimserver.geometry.GeometryRunner.run(GeometryRunner.java:197)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
~~~~

**Comment by hlg** [comment]
What OS are you using? This looks similar to the error in issue https://github.com/opensourceBIM/BIMserver/issues/633, with solution in this comment: https://github.com/opensourceBIM/BIMserver/issues/633#issuecomment-341644945.

**Comment by zaqifathis** [comment]
was your issue resolved with the solution from #633 ?
